src/components/SigninForm.jsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { login } from '../redux/users/user_reducer'
+import { Link } from 'react-router-dom'
+
+
+function SigninForm() {
+	const [formData, setFormData] = React.useState({
+		email: '',
+		password: '',
+	})
+	const dispatch = useDispatch()
+
+	const handleChange = (event) => {
+		setFormData({ ...formData, [event.target.name]: event.target.value })
+	}
+
+	const handleSubmit = (event) => {
+		event.preventDefault()
+		if (vaidateData()) {
+			dispatch(login(formData))
+		}
+	}
+
+	const vaidateData = () => {
+		if (formData.email === '' || formData.password === '') {
+			console.log('Please fill all the fields')
```

Comment:
- [ ] OPTIONAL: I can see that you are using `console.log` in order to communicate with your users about validations like "Please fill all the fields".  In the software industry, this is not a good practice because your users will almost never view the console, and makes the repo untidy.  Since your project is relying on it, I'm tagging this as optional, however, I highly suggest you use some other way to communicate with your users that does not require opening the console (like toasts, notifications, etc). 👍 

![image](https://user-images.githubusercontent.com/57421823/208410055-42ff3713-1982-4e72-ac2f-96d6d0b58425.png)